docs/ci.md

```diff
@@ -0,0 +1 @@
+../ci/README.md
```

Comment:
All markdown files in the `docs` directory must be referenced by some other referenced file or the top level `README.md`. This is mandated by the required static checks :

```
[static-checks.sh:798] ERROR: Document docs/ci.md is not referenced
```

Reply:
Thanks. fixed with the force-push.